metrics/ci_integrations/lib/integration/interface/base/party/integration_party.dart

```diff
@@ -17,4 +19,9 @@ abstract class IntegrationParty<T extends Config, K extends IntegrationClient> {
   ///
   /// Used to parse configurations related to this integration party.
   ConfigParser<T> get configParser;
+
+  /// A config validator factory for this integration party.
```

Comment:
```suggestion
  /// A configuration validator factory for this integration party.
```
Just to be consistent with `configParser` documentation.